src/amb_kf.c

```diff
@@ -22,14 +22,16 @@
 #include <cblas.h>
 #include <clapack.h>
 #include <math.h>
-#include <linear_algebra.h>
+
+#include "logging.h"
+#include "linear_algebra.h"
 #include "constants.h"
 #include "track.h"
 #include "almanac.h"
 #include "gpstime.h"
 #include "amb_kf.h"
 
-#define DEBUG_AMB_KF 0
+#define DEBUG 0
 
```

Comment:
Don't we want to define DEBUG before including logging.h? If I change this debug to 1, I think the compiler will warm me about multiple definitions of the macro.